script.js

```diff
@@ -9,123 +11,71 @@ $(document).ready(function () {
-            let instructions = response.drinks[0].strInstructions
-            // pulls ingredients from ajax 
-            let ingredients1 = response.drinks[0].strIngredient1
-            let ingredients2 = response.drinks[0].strIngredient2
-            let ingredients3 = response.drinks[0].strIngredient3
-            let ingredients4 = response.drinks[0].strIngredient4
-            // displays blank string if ingredient is equal to null
-            if (ingredients4 === null) {
-                ingredients4 = '';
-            }
-            let ingredients5 = response.drinks[0].strIngredient5
-            if (ingredients5 === null) {
-                ingredients5 = '';
-            }
-            let ingredients6 = response.drinks[0].strIngredient6
-            if (ingredients6 === null) {
-                ingredients6 = '';
-            }
-            // diplays drink name in classic-drink-title area
-            $("#classic-drink-title").text(`${name}`);
-            // creates glass name on new line
-            let glassName = $("<p>").text(`Glass: ${glass}`);
-            // creates instructions on new line
-            let drinkInstructions = $("<p>").text(`Instructions: ${instructions}`);
-            // creates ingredients on new line
-            let ingredients = $("<p>").text(`Ingredients: ${ingredients1} ${ingredients2} ${ingredients3} ${ingredients4} ${ingredients5} ${ingredients6}`);
-            // apppends all info to drinksDiv
-            drinksDiv.append(image, glassName, ingredients, drinkInstructions);
+            //           drink to display  |       title element     |     description element
+            displayDrink(response.drinks[0], $("#classic-drink-title"), $(".classic-drinks"));
         });
     }
-    // on click event for classic drink btns
-    $(document).on("click", ".drink", classicDrinkButtons);
 
+    /*
+        Both the random drink and the classic drinks have a lot of shared behavior.
+        this 'displayDrink' method encapsulates that. From what was previously here,
+        all we need are the 2 DOM elements for the drink title, and the drink description
+        area where the picture, ingredients, glass, and instructions display.
+    */
+    function displayDrink(drink, titleElement, descriptionElement) {
+        let ingredientsList = getIngredients(drink);
+        
+        const ingredientStr = ingredientsList
+            // .map(x => x.measure == null ? '' x.measure + x.ingredient) // If we want the measurement
+            .map(x => x.ingredient)
+            .join(" ");
+        
+        let randomName        = $("<p>").text(`${drink.strDrink}`);
+        let ingredients       = $("<p>").text(`Ingredients: ${ingredientStr}`);
+        let glassName         = $("<p>").text(`Glass: ${drink.strGlass}`);
+        let drinkInstructions = $("<p>").text(`Instructions: ${drink.strInstructions}`);
+        var image             = $("<img>").attr("src", drink.strDrinkThumb);
```

Comment:
Grouping similar lines of code makes sense logically, and aligning as such is pleasing to the eye for me idk :)